test/sanity/validate-modules/main.py

```diff
@@ -222,7 +222,8 @@ class ModuleValidator(Validator):
     PS_DOC_BLACKLIST = frozenset((
         'async_status.ps1',
         'slurp.ps1',
-        'setup.ps1'
+        'setup.ps1',
+        'rabbitmq_plugin.ps1'
```

Comment:
I'd like to share documentation with [rabbitmq_plugin.py](https://github.com/ansible/ansible/blob/devel/lib/ansible/modules/messaging/rabbitmq_plugin.py) module.